src/hooks/attachments/useAttachment.ts

```diff
@@ -222,10 +222,23 @@ export default function useAttachment(
     }
   }, [value])
 
+  const canDownload = React.useMemo(() => {
+    const isDownloadableAttachment = (attachment: Attachment) =>
+      (attachment.type === 'NEW' ||
+        attachment.type === 'SAVING' ||
+        !attachment.type) &&
```

Comment:
I suppose there is no reason why we should not let an attachment that is in the ERROR state be downloaded.